scss/detail.scss

```diff
@@ -0,0 +1,39 @@
+$gray : #828282;
+
+.book-container {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  padding: 15px;
+}
+
+.column {
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+}
+
+.book-title {
+  font-size: 32px;
+  font-weight: bold;
+  flex-direction: row;
+}
+
+.book-gender {
+  color: $gray;
+  font-size: 24px;
+  font-weight: bold;
+  padding: 0 0 18px 8px;
+}
+
+.book-author,
+.book-editorial,
+.book-year,
+.book-description {
+  padding: 18px 0;
+}
+
+.book-description {
+  padding: 18px 8px;
```

Comment:
Use margins instead to separate siblings, we only use padding to separate parent tags from children like so:

## HTML
```
  <div class="some-container column"> 
      <h2 class="some-title">Title</h2>
      <p>some text</p>
  </div> 
```
## CSS
```
.some-container {
  padding: (some value)px;
}

.some-title {
  margin-left: (some value)px;
}
```

Reply:
Done